packages/math/typesetter.lua

```diff
@@ -125,6 +125,12 @@ function ConvertMathML (_, content)
          SU.error("Wrong number of children in mfrac: " .. #children)
       end
       return b.fraction(children[1], children[2])
+   elseif content.command == "msqrt" then
+      local children = convertChildren(content)
+      if #children ~= 1 then
+         SU.error("Wrong number of children in msqrt")
+      end
+      return b.sqrt(children[1])
```

Comment:
Ah, I was mislead by mozilla's docs, but mathml says msqrt can have multiple children, considered in an implicit mrow. So likely we'd need to remove the condition and use: 
```
return b.sqrt(b.stackbox("H", children))
```

Reply:
Confirmed and fixed: Mozilla has even an example of it[^1], so their sentence "The square root accepts only one argument" is just imprecise. 

[^1]: Test 13 from their old [test page](https://fred-wang.github.io/MathFonts/mozilla_mathml_test/).